StreamVideoSwiftUITests/Livestreaming/LivestreamPlayer_Tests.swift

```diff
@@ -9,14 +9,18 @@ import SnapshotTesting
 import XCTest
 
 final class LivestreamPlayer_Tests: StreamVideoTestCase, @unchecked Sendable {
-
+    
     private let callId = "test"
     private let callType = "livestream"
-
+    
+    private let helper = LivestreamPlayerHelper()
+    
     @MainActor
     func test_livestreamPlayer_snapshot() async throws {
         // Given
-        let player = LivestreamPlayer(type: callType, id: callId)
+        let call = streamVideo.call(callType: callType, callId: callId)
```

Comment:
Just an FYI that you can also use `MockCall` to test further with various call states (and errors).